src/DIRAC/Core/Utilities/Extensions.py

```diff
@@ -2,14 +2,14 @@
 import argparse
 from collections import defaultdict
 import fnmatch
-from importlib import metadata
 from importlib.machinery import PathFinder
 import functools
 import importlib
 import os
 import pkgutil
 import sys
 
+import importlib_metadata as metadata
```

Comment:
Do we really need this? It seems to be there https://docs.python.org/3/library/importlib.metadata.html

Reply:
From the commit message of 6b389aa95ddfbaa97d64de92ab587be7aa19cb2e:

>  refactor: Use select instead of deprecated SelectableGroups dict interface
>
> In Python 3.8 importlib.metadata was provisional and the entry_points changed
in 3.10. To workaround this we use importlib_metadata to ensure compatibility.